src/main/java/com/stuypulse/robot/RobotContainer.java

```diff
@@ -136,7 +136,8 @@ private void configureDriverBindings() {
 
         // swerve
         driver.getLeftButton()
-            .whileTrue(new ManagerChooseScoreSide().andThen(new SwerveDriveToScorePose()));
+            .whileTrue(new SwerveDriveSlowDrive(driver));
+            // .whileTrue(new ManagerChooseScoreSide().andThen(new SwerveDriveToScorePose()));
```

Comment:
what's the align button for the future then? What if this was slow drive in cone mode and align in cube mode?

Reply:
sure that works